magenta/models/music_vae/configs.py

```diff
@@ -245,6 +245,11 @@ def update_config(config, update_dict):
     steps_per_quarter=4,
     slice_bars=16,
     gap_bars=2)
+trio_4bar_converter = data.TrioConverter(
```

Comment:
I don't think that the changes in this file are logically part of this PR, and should be upstreamed (if desired) in a separate PR.

Reply:
They were necessary to actually test the export (since this was the config used to train our model) -- but I agree technically part of the same feature